CMakeLists.txt

```diff
@@ -168,6 +168,14 @@ include( CMakePushCheckState )
 include( GNUInstallDirs )
 include( TestBigEndian )
 
+cmake_dependent_option(
+   ${_OPT}has_sentry_reporting 
+   "Has a builtin support for sending errors to Sentry"
+   On
+   "${_OPT}has_networking;DEFINED SENTRY_DSN_KEY;DEFINED SENTRY_HOST;DEFINED SENTRY_PROJECT"
```

Comment:
I'm trying to build the new code, without success yes
I tried
`cmake -GXcode -Daudacity_has_sentry_reporting=YES -Daudacity_has_networking=YES -DSENTRY_DSN_KEY=YES -DSENTRY_HOST=YES _DSENTRY_PROJECT=YES ..`